plugins/bbcode.md

```diff
@@ -57,6 +56,6 @@ tinymce.init({
 });
 ```
 
-## Live example
+## Interactive example
```

Comment:
In my latest PR I used "Live" example, it seems this is a bit inconsistent

Reply:
We're switching to this, 'live' is a bit vague.